lib/knapsack/presenter.rb

```diff
@@ -51,10 +51,15 @@ def time_offset_warning
         }
         if Knapsack.tracker.time_exceeded?
           str << %{
-Tests on this CI node took more than max allowed node time execution.
+Tests on this CI node took more than the max allowed node time execution.
 Please regenerate your knapsack report.
-If that didn't help then split your heavy test file
-or bump time_offset_in_seconds setting.}
+
+If that didn't help then split your slow test file into smaller test files
```

Comment:
Consider: 
`If that doesn't help, you can split your slowest test files into smaller files, or bump up the time_offset_in_seconds setting.`